ecosystem/plugin-docsearch/src/client/components/Docsearch.ts

```diff
@@ -1,14 +1,10 @@
-import { default as docsearch } from '@docsearch/js'
-import { usePageLang, useRouteLocale } from '@vuepress/client'
-import { isArray } from '@vuepress/shared'
-import { computed, defineComponent, h, onMounted, watch } from 'vue'
+import { defineComponent, h } from 'vue'
 import type { PropType } from 'vue'
 import type { DocsearchOptions } from '../../shared/index.js'
-import { useDocsearchShim } from '../composables/index.js'
+import { useDocsearch } from '../composables/index.js'
+import { options, searchButtonTemplate } from '../utils/index.js'
 
 declare const __DOCSEARCH_INJECT_STYLES__: boolean
-declare const __DOCSEARCH_OPTIONS__: DocsearchOptions
-const options = __DOCSEARCH_OPTIONS__
 
 if (__DOCSEARCH_INJECT_STYLES__) {
```

Comment:
duplicated?

Reply:
It's a Boolean value, to ensure the code can be treeshaken, I would prefer keeping it.

`if (false)` can be removed all times, but not

```ts
const a = 1;

if (a) {
 // xxx
}
```